### ISSUE TITLE
A lot of forests are missing from the map

### ISSUE THREAD

**Issue by pronvit** [osmandapp/OsmAnd-iOS#4055]
Most of the state forests in QLD, Australia (and probably in other places?) are missing from the map in OsmAnd. See the screenshots from OSM web map and OsmAnd. This happens with any map style and details settings. I tried debugging and looks like they're not in a list of objects for a tile at all but I'm not too familiar with osmand core code. On the other hand, if you try searching for say Benarkin State Forest, it will be found (and you'll be taken to an invisible object on the map), so it seems to be present in the map data.

<img width="1204" alt="Screenshot 2024-10-13 at 9 42 56 am" src="https://github.com/user-attachments/assets/e60e0425-52f7-4d7b-a2df-e2a1ee865f9b">

<img width="935" alt="Screenshot 2024-10-13 at 9 45 52 am" src="https://github.com/user-attachments/assets/8d728120-c66d-4fb5-98d8-39be8486522a">

**Comment by vshcherb** [comment]
https://www.openstreetmap.org/relation/8681576#map=11/-26.8225/152.1885

**Comment by vshcherb** [comment]
OsmAnd doesn't display type=boundary and only type=multipolygon, so even though default rendering is using it I think it's a mapping issue. If it's that area actually is the forest, then it shouldn't be mapped as boundary

**Comment by pronvit** [comment]
Yeah, the type did seem strange to me. Is there a quick and dirty solution for me given I'm using a custom build already anyway? There's a ton of objects like this around here, quite possibly they all are a result of an incorrect import or smth, but on the other hand looks like it's only OsmAnd that doesn't render them.

**Comment by pronvit** [comment]
Sure I get that, but I meant could you point me in the right direction how to fix this locally for myself? I can't seem to get these objects to show no matter how I tweak the style (while I expected to be able to render them at least like the similar nat parks boundaries), do they get filtered out somewhere in the code or even during map creation?

**Comment by vshcherb** [comment]
It's hard coded, for protected_area there was an exception cause we wanted to render as transparent shader so from line it converted to polygon

**Comment by pronvit** [comment]
Alright, I still couldn't find where protected_area gets a special treatment, but I added type conversion for forests to rendering_types.xml, recreated the map form osm, and that did the trick.
